controls/roles/manage-service/tasks/main.yml

```diff
@@ -98,7 +98,7 @@
         hex_chars: '0123456789abcdef'
     become: yes
     when:
-      - stereum_service_configuration.service in ['BesuService', 'GethService', 'NethermindService', 'ErigonService', 'LighthouseBeaconService', 'NimbusBeaconService', 'PrysmBeaconService', 'TekuBeaconService', 'LodestarBeaconService']
+      - stereum_service_configuration.service in ['BesuService', 'GethService', 'NethermindService', 'ErigonService', 'LighthouseBeaconService', 'NimbusBeaconService', 'PrysmBeaconService', 'TekuBeaconService', 'LodestarBeaconService', 'NethermindGnosisService']
```

Comment:
dont make a new `NethermindGnosisService`. Take a `NethermindService` and change the network to xdai

Reply:
done!